From 93898dca2fe6d2aeefa343b723703d0505563340 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 18 Dec 2023 15:15:17 +0100
Subject: [PATCH 0233/1695] dts: stm32: add cryp1 as an ETZPC subnode

Similarly to the core/arch/arm/dts/stm32mp15xf.dtsi file, the CRYP1
should be a subnode of the ETZPC.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 7fec47b22f81fa9eab03b5111a51cb0a5ab4a21a dts: stm32: add cryp1 as an ETZPC subnode]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I5820729c92324abe6b3a9f1aa74e95358973b71a
---
 core/arch/arm/dts/stm32mp15xc.dtsi | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp15xc.dtsi b/core/arch/arm/dts/stm32mp15xc.dtsi
index b06a55a2f..8fb63a638 100644
--- a/core/arch/arm/dts/stm32mp15xc.dtsi
+++ b/core/arch/arm/dts/stm32mp15xc.dtsi
@@ -4,15 +4,15 @@
  * Author: Alexandre Torgue <alexandre.torgue@st.com> for STMicroelectronics.
  */
 
-/ {
-	soc {
-		cryp1: cryp@54001000 {
-			compatible = "st,stm32mp1-cryp";
-			reg = <0x54001000 0x400>;
-			interrupts = <GIC_SPI 79 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&rcc CRYP1>;
-			resets = <&rcc CRYP1_R>;
-			status = "disabled";
-		};
+#include "stm32mp15xa.dtsi"
+
+&etzpc {
+	cryp1: cryp@54001000 {
+		compatible = "st,stm32mp1-cryp";
+		reg = <0x54001000 0x400>;
+		interrupts = <GIC_SPI 79 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&rcc CRYP1>;
+		resets = <&rcc CRYP1_R>;
+		status = "disabled";
 	};
 };
-- 
2.47.2

